src/components/material/material.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import clsx from "clsx";
+import { useGetCoverCollection } from "../../core/cover-service-api/cover-service";
+
+export type MaterialProps = {
+  animate: boolean;
+  size: "small" | "medium" | "large" | "original";
+  tint?: "20" | "40" | "60" | "80" | "120";
+  materialId: string;
+  materialDescription?: string;
+  materialUrl?: string;
+};
+
+export const Material = ({
+  materialUrl,
+  materialDescription,
+  size,
+  animate,
+  tint,
+  materialId
+}: MaterialProps) => {
+  const { data } = useGetCoverCollection({
+    type: "pid",
+    identifiers: [materialId],
+    sizes: [size]
+  });
+
+  type TintClassesType = {
+    [key: string]: string;
+  };
+  const tintClasses: TintClassesType = {
+    default: "bg-identity-tint-120",
+    "120": "bg-identity-tint-120",
+    "80": "bg-identity-tint-80",
+    "60": "bg-identity-tint-60",
+    "40": "bg-identity-tint-40",
+    "20": "bg-identity-tint-20"
+  };
+
+  const classes = {
+    wrapper: clsx(
+      `material material--${size}`,
+      tintClasses[tint || "default"],
+      {
+        material__animate: animate
+      }
+    )
+  };
+
+  const coverUrl = data?.[0]?.imageUrls?.[`${size}`]?.url;
+  const materialCover = coverUrl && (
```

Comment:
This is a tricky one, as if the img has alt="" and is inside a link that doesnt have any text inside, it will cause an error with the accessibility-tool digitaliseringsstyrelsen are using (sorry for the danglish). Do we have a material title as an alternative instead?

Reply:
Unfortunately, we do not have a title unless we send one from a component above. 
Do you think we should do it?

Reply:
This element will probably show up a lot, which could cause a lot of accessibility errors, so I think something should be done. The title would be good if it is possible

Reply:
We have changed it so that there must be a description before an alt text before a tag will be rendered

Reply:
It has been changed so that there must be a description for it to be rendered a tag. https://github.com/danskernesdigitalebibliotek/dpl-react/pull/42/commits/5c47757ab7fb6783408db96a4160ac9ab1a8f647